taca_ngi_pipeline/deliver/deliver_mosler.py

```diff
@@ -0,0 +1,292 @@
+""" 
+    Module for controlling deliveries os samples and projects to Mosler (THE MOSLER!!!!)
+"""
+
+import paramiko
+import getpass
+import tarfile
+import threading
+import glob
+import time
+
+
+from deliver import *
+
+class MoslerDeliverer(Deliverer):
+
+    def __init__(self, projectid=None, sampleid=None, **kwargs):
+        super(MoslerDeliverer, self).__init__(
+            projectid,
+            sampleid,
+            **kwargs)
+        self.moslerdeliverypath = getattr(self, 'moslerdeliverypath', None)
+        self.moslersftpserver = getattr(self, 'moslersftpserver', None)
+        self.moslersftpserver_user = getattr(self, 'moslersftpserver_user', None)
+        self.moslersftpmaxfiles = getattr(self, 'moslersftpmaxfiles', None)
+
+
+class MoslerProjectDeliverer(MoslerDeliverer):
+    """ This object takes care of delivering project samples to mosler.
+        When delivering to Mosler the delivery can take place only at project level
+    """
+    def __init__(self, projectid=None, sampleid=None, **kwargs):
+        super(MoslerProjectDeliverer, self).__init__(
+            projectid,
+            sampleid,
+            **kwargs)
+    
+    def all_samples_delivered(
+            self,
+            sampleentries=None):
+        """ Checks the delivery status of all project samples
+
+            :params sampleentries: a list of sample entry dicts to use instead
+                of fetching from database
+            :returns: True if all samples in this project has been successfully
+                delivered, False otherwise
+            THIS HAS BEEN COPIED AND PASTED FROM deliver.py ProjectDeliverer class
```

Comment:
Couldn't you make `MoslerProjectDeliverer` a subclass of `ProjectDeliverer` instead?